XenAdmin/Wizards/GenericPages/SelectMultipleVMDestinationPage.cs

```diff
@@ -134,14 +135,17 @@ public IXenObject SelectedTarget
             get => _selectedTarget;
             set
             {
-                var oldTarget = _selectedTarget;
                 _selectedTarget = value;
-
-                if (oldTarget?.opaque_ref != SelectedTarget?.opaque_ref)
-                    OnSelectedTargetChanged();

```

Comment:
I have removed this because I wanted the event to fire even if the user selected the last target.

This is because if a user first selects one host for a VM mapping and then re-selects the same host for another VM mapping, the event wasn't fired (and I couldn't update `AllSelectedTargets`)